tests/kuttl/test-suites/default/deps/infra.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  storageClass: crc-csi-hostpath-provisioner
+  mariadb:
+    enabled: false
+  galera:
+    enabled: true
+    templates:
+      openstack:
+        storageRequest: 500M
+        storageClass: crc-csi-hostpath-provisioner
+      openstack-cell1:
+        storageRequest: 500M
+        storageClass: crc-csi-hostpath-provisioner
+  rabbitmq:
+    templates:
+      rabbitmq:
+        replicas: 1
+      rabbitmq-cell1:
+        replicas: 1
```

Comment:
same for rabit so this can also be removed.

if we wanted to run watcher on its own rabbit or galera instance this is how we would create the instance and then we would just pass the name to use to the watcher cr later.